pymc_marketing/mmm/preprocessing.py

```diff
@@ -32,7 +32,7 @@ class MaxAbsScaleTarget:
 
     @preprocessing_method_y
     def max_abs_scale_target_data(self, data: pd.Series) -> pd.Series:
-        target_vector = data.reshape(-1, 1)
```

Comment:
Based on the deprecation #406 

However, is this the correct signature of this function. Could this data also be a `numpy.array`?